DesktopEdge/MainWindow.xaml.cs

```diff
@@ -1723,6 +1723,10 @@ private void MainUI_Deactivated(object sender, EventArgs e) {
             }
         }
 
+        private void Label_MouseDoubleClick(object sender, MouseButtonEventArgs e) {
```

Comment:
Instead of just deleting this function, adding it back.  It is never referenced anywhere else.